apollo/lib/schemas/agent.py

```diff
@@ -22,7 +23,15 @@ def name_must_not_exist(cls, name, **kwargs):
         raise ValueError("An agent with the given name already exists")
 
 
+class ConnectionStateEnum(str, Enum):
```

Comment:
If possible I'd use the [WebSocketStateEnum](https://github.com/encode/starlette/blob/master/starlette/websockets.py#L9) rather than defining your own. You could always post process to lower the string if you want to.